# HOXB13_project
DESeq2_example.R:   R code example to do Gene-level differential expression analysis with DESeq2 for RNA-seq NGS data (e.g. fig1A and figS1A genes)

Hist.R and MakeCurve2PDF.pl:  R/perl code example to draw intensity plots (e.g. fig3E and fig3I)

MA_plot_example.R:  R code example to draw MA plots (e.g. fig3C and figS3C)

NGS_pipeline_example.txt:   pipeline to process ChIP-seq and RNA-seq NGS data. BETA command for fig1G and deepTools commands to plot heatmaps (e.g. fig1H, fig3B and fig3D)

WCDT_correlation_plot.R:  R code example to draw corrleation plot with WCDT methylation data which were downloaded from dbGaP (e.g. figS7E)

WCDT_scatter_plot.R:  R code example to draw scatter plot with WCDT methylation data which were downloaded from dbGaP (e.g. fig5G)

bismark_pipeline.txt:   pipeline to process WCDT methylation data which were downloaded from dbGaP using bismark software

convert_counts_to_FPKM_igenome_gene.pl: perl code example to convert raw counts to RPKM/FPKM values

gene_correlation_for_all_tables_4ppt_ggplot.R:  R code example to draw scatter plots with prostate cancer patient data (e.g. figS7F and figS7G)

gene_expression_for_all_tables_ggboxplot.R:   R code example to draw boxplots with prostate cancer patient data (e.g. figS9A)

methylation_counts.pl:  perl code example to calculate mythylation rates from the coverate file which were generated by bismark

pheatmap_example.R:   R code example to draw heatmaps using pheatmap package (e.g. fig4D and fig4J)

venn_diagram_example.R:   R code example to draw venn diagrams using Vennerable package (e.g. fig1I, fig3A, fig3I, figS3A and figS4B)
