# HOXB13_project
DESeq2.R:   R code to do gene-level differential expression analysis with DESeq2 for RNA-seq NGS data (fig1a and extended fig1a)

Hist.R and MakeCurve2PDF.pl:  R/perl code to draw intensity plots (fig3d and extended fig3c)

MA_plot.R:  R code to draw MA plots (extended fig3a and extended fig3b)

NGS_pipeline.txt:   pipeline to process ChIP-seq and RNA-seq NGS data. BETA command (fig1g) and deepTools commands to plot heatmaps (e.g. fig1h, fig3B and fig3D)

WCDT_correlation_plot.R:  R code to draw corrleation plot with WCDT methylation data which were downloaded from dbGaP (e.g. figS7E)

WCDT_scatter_plot.R:  R code to draw scatter plot with WCDT methylation data which were downloaded from dbGaP (e.g. fig5G)

bismark_pipeline.txt:   pipeline to process WCDT methylation data which were downloaded from dbGaP using bismark software

convert_counts_to_FPKM_igenome_gene.pl: perl code to convert raw counts to RPKM/FPKM values

gene_correlation_for_all_tables_4ppt_ggplot.R:  R code to draw scatter plots with prostate cancer patient data (e.g. figS7F and figS7G)

gene_expression_for_all_tables_ggboxplot.R:   R code to draw boxplots with prostate cancer patient data (e.g. fig5A and figS9A)

methylation_counts.pl:  perl code to calculate mythylation rates from the coverate file which were generated by bismark

pheatmap.R:   R code to draw heatmaps using pheatmap package (e.g. fig4D and fig4J)

venn_diagram.R:   R code to draw venn diagrams using Vennerable package (e.g. fig1I, fig3A, fig3I, figS3A and figS4B)
