# HOXB13_project
DESeq2.R:   R code to do gene-level differential expression analysis with DESeq2 for RNA-seq NGS data (Fig.1a and extended data Fig.1a)

Hist.R and MakeCurve2PDF.pl:  R/perl code to draw intensity plots (Fig.3d and extended data Fig.3c)

MA_plot.R:  R code to draw MA plots (extended data Fig.3a,b)

NGS_pipeline.txt:   pipeline to process ChIP-seq and RNA-seq NGS data. BETA command (Fig.1g) and deepTools commands to plot heatmaps (Fig.1h and Fig.3b,c)

WCDT_correlation_plot.R:  R code to draw corrleation plot with WCDT methylation data which were downloaded from dbGaP (Fig.5f)

WCDT_scatter_plot.R:  R code to draw scatter plot with WCDT methylation data which were downloaded from dbGaP (Fig.5e)

bismark_pipeline.txt:   pipeline to process WCDT methylation data which were downloaded from dbGaP using bismark software

convert_counts_to_FPKM_igenome_gene.pl: perl code to convert raw counts to RPKM/FPKM values

gene_correlation_for_all_tables_4ppt_ggplot.R:  R code to draw scatter plots with prostate cancer patient data (supplementary Fig.5b,c)

gene_expression_for_all_tables_ggboxplot.R:   R code to draw boxplots with prostate cancer patient data (Fig.5a and extended data Fig.8a)

methylation_counts.pl:  perl code to calculate mythylation rates from the coverate file which were generated by bismark

pheatmap.R:   R code to draw heatmaps using pheatmap package (Fig4c,h)

venn_diagram.R:   R code to draw venn diagrams using Vennerable package (Fig.1i,3a, extended data Fig.3c,4a and supplementary Fig.2a)

input folder:   Figure input text files

result folder:   Figure results
